core/runner_actor.py

```diff
@@ -0,0 +1,102 @@
+# Add the parent directory to the PATH to allow imports.
+import os, sys, inspect
+currentdir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+parentdir = os.path.dirname(currentdir)
+sys.path.insert(0,parentdir)
+import logging
+import random
+import uuid
+import ray
+
+from core.runner import DMLRunner
+from custom.keras import model_from_serialized, get_optimizer
+from data.iterators import count_datapoints
+from data.iterators import create_train_dataset_iterator
+from data.iterators import create_test_dataset_iterator
+from core.utils.keras import train_keras_model, validate_keras_model
+from core.utils.keras import serialize_weights
+
+logging.basicConfig(level=logging.DEBUG,
+                    format='[Runner] %(asctime)s %(levelname)s %(message)s')
+
+@ray.remote
+class DMLRunnerActor(DMLRunner):
+    """
+    DML Runner Actor
+
+    NOTE: This is a wrapper around DMLRunner so that testing on DMLRunner can be 
+    independent of interaction with Ray
+
+    NOTE 2: Please read https://ray.readthedocs.io/en/latest/actors.html. This 
+    class serves as an actor, where methods of each instance are run in parallel 
+    across instances. This is so that runners can run jobs independently of each
+    other.
+
```

Comment:
No need for the docs in lines 35-43. We wouldn't want to maintain docs in the original class and the actor.

Reply:
Removed lines 35-43. The method comments are copied exactly from `DMLRunner`, are you sure you still want me to delete them?